dataprofiler/labelers/char_load_tf_model.py

```diff
@@ -227,7 +237,7 @@ def _construct_model(self):
         default_ind = self.label_mapping[self._parameters["default_label"]]
         model_loc = self._parameters["model_path"]
 
-        self._model = tf.keras.models.load_model(model_loc)
+        self._model: tf.keras.Model = tf.keras.models.load_model(model_loc)
         softmax_output_layer_name = self._model.outputs[0].name.split("/")[0]
         softmax_layer_ind = labeler_utils.get_tf_layer_index_from_name(
```

Comment:
would it make more sense to cast `softmax_layer_ind` to int at assignment rather than down below

Reply:
good call out